jpo-ode-core/src/main/resources/schemas/schema-bsm.json

```diff
@@ -546,17 +528,7 @@
                                                                 "type": "boolean"
                                                             }
                                                         },
-                                                        "required": [
-                                                            "lowBeamHeadlightsOn",
-                                                            "highBeamHeadlightsOn",
-                                                            "leftTurnSignalOn",
-                                                            "rightTurnSignalOn",
-                                                            "hazardSignalOn",
-                                                            "automaticLightControlOn",
-                                                            "daytimeRunningLightsOn",
-                                                            "fogLightOn",
-                                                            "parkingLightsOn"
-                                                        ],
```

Comment:
This is a bitstring too.

Reply:
Opening a work item that will reference this.